python-backend/app/api/permits/namespace/permits.py

```diff
@@ -1,24 +1,26 @@
 from flask_restplus import Namespace
 
-from ..permit.resources.permit import PermitResource
+from ..permit.resources.permit import PermitResource, PermitListResource
 from ..permit.resources.permit_status_code import PermitStatusCodeResource
-from ..permit_amendment.resources.permit_amendment import PermitAmendmentResource
-from ..permit_amendment.resources.permit_amendment_document import PermitAmendmentDocumentResource
+from ..permit_amendment.resources.permit_amendment import PermitAmendmentResource, PermitAmendmentListResource
+from ..permit_amendment.resources.permit_amendment_document import PermitAmendmentDocumentListResource, PermitAmendmentDocumentResource
 
 api = Namespace('permits', description='Permit related operations')
```

Comment:
permits should be moved under mines.... this might be the time to move the routes. we can move the files on sprint planning day.

Reply:
Moved routes and files under mines.